framework/include/kernels/BodyForce.h

```diff
@@ -44,4 +44,15 @@ class BodyForce : public Kernel
 
   /// Optional Postprocessor value
   const PostprocessorValue & _postprocessor;
+
+  using GenericKernel<is_ad>::_qp;
+  using GenericKernel<is_ad>::_i;
+  using GenericKernel<is_ad>::_t;
+  using GenericKernel<is_ad>::_test;
```

Comment:
I don't really like that this requires a `using` in all the classes inherited from `GenericKernel`...